csharp/src/Ice/Network.cs

```diff
@@ -161,13 +153,7 @@ internal static Socket CreateSocket(bool udp, AddressFamily family)
                     throw new DNSException(host);
                 }
 
-                IEnumerable<IPEndPoint> addrs = addresses;
-                if (selType == EndpointSelectionType.Random)
-                {
-                    addrs = addrs.Shuffle();
```

Comment:
Since we no longer Shuffle anything, we should eliminate EnumerableExtentions. We can always restore it if we need it again.